app/_src/policies/introduction.md

```diff
@@ -1,32 +1,30 @@
 ---
 title: Policies
 ---
-
-{% tip %}
-**Need help?** Installing and using {{site.mesh_product_name}} should be as easy as possible. [Contact and chat](/community) with the community in real-time if you get stuck or need clarifications. We are here to help.
```

Comment:
why is the whole tip removed? does not seem like this is not link related

Reply:
It's useless and hard to maintain